packages/http/src/mocker/generator/__tests__/JSONSchema.spec.ts

```diff
@@ -80,12 +83,13 @@ describe('JSONSchema generator', () => {
       };
 
       it('will have a string property matching the ip regex', () => {
-        const instance = generate(schema);
-        expect(instance).toHaveProperty('ip');
-        const ip = get(instance, 'ip');
+        assertRight(generate(schema), instance => {
```

Comment:
Since `generate` return Either we should have a test of failure. Maybe a circular object will do the job?

Reply:
ekhm :P

Reply:
Ahi, I didn't see this. Will add.